apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -728,8 +733,28 @@ class TopInstructions extends Component {
               />
             )}
             {this.isViewingAsTeacher &&
-              (hasContainedLevels || teacherMarkdown) && (
+              (hasContainedLevels ||
+                teacherMarkdown ||
+                exampleSolutions.length > 0) && (
                 <div>
+                  {exampleSolutions.length > 0 && (
+                    <div style={styles.exampleSolutions}>
+                      {exampleSolutions.map((example, index) => (
+                        <Button
+                          __useDeprecatedTag
```

Comment:
Curious as to why we're using the deprecated version of the button here?

Reply:
Because I forgot to remove this in copying it over 🙃 .